tests/security/cc/misc.pm

```diff
@@ -13,11 +13,12 @@ use warnings;
 use testapi;
 use utils;
 use audit_test qw(run_testcase compare_run_log);
+use Utils::Architectures qw(is_s390x);
 
 sub run {
     my ($self) = shift;
 
-    reset_consoles;
+    reset_consoles if !is_s390x;
```

Comment:
For the future let me please provide some humble suggestions:
1. The git commit message should explain the why rather than the why. The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see https://commit.style/ or http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.
2. It's a better practice to avoid any architecture specific calls in test modules which should be architecture-agnostic, this module included. The way to do it would be to consider extending `reset_consoles` itself to  do what you need instead of just doing some special in this test module for unclear reason
3. better use `function unless is_s390x` instead of `function if !is_s390x` :)

Reply:
>     2. this issue began on the 5th of December and only affects the `!s390x` architectures. Notably, this module has not been modified for a long time, so it is highly likely that some other external change (the specifics of which are currently unknown) is the root cause. Given the team's limited capacity, we are unable to investigate every "external factor issue". Therefore, the fastest and, in my opinion, the clearest approach to restore the module's functionality was to implement this fix.

It's still a recipe for failure later on. Think about anyone reading this code in some months because there is a test failure or a need to change the test module. How would they possibly be able to understand why there is a line like that